runtime/CMakeLists.txt

```diff
@@ -573,7 +573,7 @@ endif()
 if(J9VM_MODULE_THREAD)
 	add_subdirectory(thread)
 endif()
-if(J9VM_INTERP_NATIVE_SUPPORT AND NOT OMR_ARCH_RISCV)
+if(J9VM_INTERP_NATIVE_SUPPORT AND NOT (OMR_ARCH_AARCH64 AND OMR_OS_OSX))
```

Comment:
This change looks obsolete, we have JIT for aarch64 mac now.

Reply:
Ah, I did not noticed! Fixed in new version (f070c91)